pyproject.toml

```diff
@@ -1,5 +1,157 @@
+[project]
```

Comment:
I don't know if it is worthy the trouble, or if it fits the example, but PyPA also maintains a live version of the spec now:

```suggestion
[project]
# A general reference for all the fields available in `[project]` can be found at:
# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/

```